test/Sync.t.sol

```diff
@@ -137,95 +129,18 @@ contract SyncTest is Test, Deployers, GasSnapshot {
-        // Sync has been called.
-        assertEq(manager.getReserves(currency3), type(uint256).max);
-
-        uint256 maxBalanceCurrency3 = uint256(int256(type(int128).max));
-
-        Actions[] memory actions = new Actions[](10);
-        bytes[] memory params = new bytes[](10);
-
-        // 1. First supply a large amount of currency3 to the pool, by minting and transfering.
-        // Encode a MINT.
-        actions[0] = Actions.MINT;
-        params[0] = abi.encode(address(this), currency3, maxBalanceCurrency3);
-
-        // Encode a TRANSFER.
-        actions[1] = Actions.TRANSFER_FROM;
-        params[1] = abi.encode(currency3, address(this), address(manager), maxBalanceCurrency3);
-
-        // Encode a SETTLE.
-        actions[2] = Actions.SETTLE;
-        params[2] = abi.encode(currency3);
-
-        // 2. Second check that the balances (token balances, reserves balance, and delta balance) are as expected.
-        // The token balance of the pool should be the full balance.
-        // The reserves balance should have been updated to the full balance in settle.
-        // And the delta balance should be 0, because it has been fully settled.
-
-        // Assert that the manager balance is the full balance.
-        actions[3] = Actions.ASSERT_BALANCE_EQUALS;
-        params[3] = abi.encode(currency3, address(manager), maxBalanceCurrency3);
-
-        // Assert that the reserves balance is the full balance.
-        actions[4] = Actions.ASSERT_RESERVES_EQUALS;
-        params[4] = abi.encode(currency3, maxBalanceCurrency3);
-
-        // Assert that the delta is settled.
-        actions[5] = Actions.ASSERT_DELTA_EQUALS;
-        params[5] = abi.encode(currency3, address(router), 0);
-
-        // 3. Take the full balance from the pool, but do not call sync.
-        // Thus reservesBefore stays > 0. And the next reserves call will be 0 causing a revert.
-
-        // Encode a TAKE.
-        actions[6] = Actions.TAKE;
-        params[6] = abi.encode(currency3, address(this), maxBalanceCurrency3);
-
-        // Assert that the actual balance of the pool is 0.
-        actions[7] = Actions.ASSERT_BALANCE_EQUALS;
-        params[7] = abi.encode(currency3, address(manager), 0);
-
-        // Assert that the reserves balance is the old pool balance because sync has not been called.
-        actions[8] = Actions.ASSERT_RESERVES_EQUALS;
-        params[8] = abi.encode(currency3, maxBalanceCurrency3);
-
-        // Encode a SETTLE.
-        actions[9] = Actions.SETTLE;
-        params[9] = abi.encode(currency3);
-
-        // Expect an underflow/overflow because reservesBefore > reservesNow since sync() had not been called before settle.
-        vm.expectRevert(abi.encodeWithSignature("Panic(uint256)", 0x11));
-
-        router.executeActions(actions, params);
-    }
-
     // @notice This tests expected behavior if you DO NOT call sync. (ie. Do not interact with the pool manager properly. You can lose funds.)
```

Comment:
Can you update the comment and test name - I feel this is out of date? And it would be nice to have a new summary of whats being tested

Reply:
Updated the comment and the name, also added some comments in the test to clarify whats is happening!